site/docs/v1/tech/client-libraries/netcore.adoc

```diff
@@ -37,11 +37,100 @@ include::docs/v1/tech/client-libraries/_static-patch-note.adoc[]
 :client_library_language!:
 
 
+== FusionAuth Templates for .NET
+
+FusionAuth has a collection of .NET templates available as a NuGet package. Quickly create new secured .NET applications with FusionAuth templates using the .NET CLI or Visual Studio. The new applications are pre-configured to use FusionAuth for authentication and authorization. Only standard .NET security libraries are used in the templates.
+
+Currently, the following .NET templates are implemented:
+
+* FusionAuth Blazor Server Application.
+* FusionAuth MVC Application.
+* FusionAuth Web API Application.
+
+
+=== Requirements
+
+* A FusionAuth installation. You can link:/docs/v1/tech/installation-guide/[install FusionAuth locally, or sign up for a hosted account].
+* .NET Core 7.0 SDK or higher.
+* Visual Studio 2022 for Mac, version 17.6 and above (optional).
+* Visual Studio 2022 for Windows, version 17.6 and above (optional).
+
+
+=== Installing the Templates
+
+FusionAuth .NET templates are available on link:https://www.nuget.org/packages/FusionAuth.Templates/[NuGet]. You can install them by running the following command in your terminal.
+
+```bash
+dotnet new install FusionAuth.Templates::1.0.0
+```
+
+When installed successfully, the templates will be available in the .NET CLI and Visual Studio. The installation is the same for both Windows and macOS.
+
+=== Using the Templates
+
+To help you set up a valid application in FusionAuth, we have created a .NET FusionAuth link:/docs/v1/tech/installation-guide/kickstart[Kickstart] script, available on https://github.com/FusionAuth/fusionauth-example-client-libraries/tree/main/dotnet/Kickstart[GitHub]. Follow the instructions in the Github repo to set up FusionAuth for your project. If you don't want to use the Kickstart script, you can set up your application manually in FusionAuth, using the values referenced in the Kickstart script. 
```

Comment:
```suggestion
To help you set up a valid application in FusionAuth, we have created a .NET FusionAuth link:/docs/v1/tech/installation-guide/kickstart[Kickstart], available on https://github.com/FusionAuth/fusionauth-example-client-libraries/tree/main/dotnet/Kickstart[GitHub]. Follow the instructions in the Github repo to set up FusionAuth for your project. If you don't want to use the Kickstart script, you can set up your application manually in FusionAuth, using the values referenced in the Kickstart.json file. 
```

also should link directly to the Kickstart.json file in it's final location

Reply:
I don't see the changes for https://github.com/FusionAuth/fusionauth-example-client-libraries/tree/main/dotnet/Kickstart.  Will that come later?